cliquet/events.py

```diff
@@ -41,25 +41,47 @@ def __init__(self, action, timestamp, read_records, request):
 
 
 class ResourceChanged(_ResourceEvent):
-    """Triggered when a resource is changed.
+    """Triggered when a resource is being changed.
```

Comment:
Or if we want to keep it, naming it `BeforeResourceChanged` might convey more meaning.